config/webpack.js

```diff
@@ -20,13 +20,15 @@ const pages = {
 
 const htmlHeadTemplate = _.template(fs.readFileSync('./html/includes/_head.ejs', 'utf8'));
 const htmlFooterTemplate = _.template(fs.readFileSync('./html/includes/_footer.ejs', 'utf8'));
+const htmlIconsTemplate = _.template(fs.readFileSync('./html/includes/_icons.ejs', 'utf8'));
```

Comment:
This is a static file, no need to import it as a template. Just have a .svg file and import the raw text as is in the template.